### PR TITLE
Feature/vmo 6309/fix get property action

### PR DESCRIPTION
**Pull request by bulatgab**
Fix issues that I had in voto5:
- turn `block_getContactPropertyOnIndex` and `block_getContactPropertyValueByKey` into getters which are synchronous. (They were actions and were returning promises. We couldn't await there)
- use double arrow function because we need to provide params (with regular getters we don't use params). This is called [method-style access](https://vuex.vuejs.org/guide/getters.html#method-style-access)

### CODE REVIEW

File: src/store/flow/block.ts [hunk 1]
```diff
@@ -266,7 +268,10 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
    * - Valid
    * - Invalid
    */
-  async block_updateBranchingExitsWithInvalidScenario({state, dispatch}, {blockId, test}: {blockId: IBlock['uuid'], test: IBlockExit['test']}) {
+  async block_updateBranchingExitsWithInvalidScenario(
+    {state, dispatch},
+    {blockId, test}: {blockId: IBlock['uuid'], test: IBlockExit['test']},
+  ) {
```
Comment (bulatgab) [inline src/store/flow/block.ts:274]: eslint fix

File: src/store/flow/block.ts [hunk 2]
```diff
@@ -8,14 +8,16 @@ import {
 } from '@floip/flow-runner'
 import {ActionTree, GetterTree, MutationTree} from 'vuex'
 import {IRootState} from '@/store'
-import {defaults, get, has, isArray, isNil, last, reject, snakeCase, toPath} from 'lodash'
+import {defaults, has, isArray, isNil, last, reject, snakeCase} from 'lodash'
```
Comment (bulatgab) [inline src/store/flow/block.ts:11]: eslint fix

File: src/store/flow/block/set-contact-property.ts [hunk 3]
```diff
@@ -54,90 +72,71 @@ export const mutations: MutationTree<IFlowsState> = {
 }
 
 export const actions: ActionTree<IFlowsState, IRootState> = {
-    /**
+  /**
    * Make sure we have Array set_contact_property,
    * if it's not an array, then the mutation will set an object like {set_contact_property: {0: object1, 1: object2}} instead
    */
```
Comment (bulatgab) [inline src/store/flow/block/set-contact-property.ts:78]: 👇 fix indentation

**Review by object-required — APPROVED** [review]
I didn't know component getters cannot be async 🤔 
LGTM 👍

**Pull request merged**